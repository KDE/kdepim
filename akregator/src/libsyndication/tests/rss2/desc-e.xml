<?xml version="1.0" encoding="iso-8859-1"?>
<!-- taken from http://virtuelvis.com/archives/2004/03/11-ways-to-valid-rss -->
<!-- 
    This is a RSS 2.0 test feed part of a suite created with the purpose of identifying various ways of specifying descriptions and full body content

    Each and every feed in this suite contains only one item, and although
    these items claim to have a link/permalink, the permalink does not exist

    Test E:  Plain-text with inline HTML using escaping.
-->
<rss version="2.0">
  <channel>
    <title>RSS 2.0 Content test E</title>
    <link>http://www.example.com/</link>
    <description>Feed Description</description>
    <lastBuildDate>Sun, 14 Mar 2004 19:59:12 +0100</lastBuildDate>
    <pubDate>Sun, 14 Mar 2004 19:59:12 +0100</pubDate>    
    <item>
      <title>Plain-text with inline HTML using escaping</title>
      <link>http://www.example.com/a/1</link>
      <guid>http://www.example.com/a/1</guid>
      <pubDate>Sun, 14 Mar 2004 19:59:12 +0100</pubDate>

      <description>This is supposed to be a paragraph. With a &lt;a href="http://www.example.com/"&gt;link inside&lt;/a&gt;.
      
      Another paragraph, with &lt;a href="http://www.example.com/"&gt;yet another link&lt;/a&gt;.</description>
    </item>
  </channel>
</rss>