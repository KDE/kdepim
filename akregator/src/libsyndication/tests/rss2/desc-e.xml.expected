# Feed begin ######################
title: #RSS 2.0 Content test E#
link: #http://www.example.com/#
description: #Feed Description#
# Item begin ######################
id: #http://www.example.com/a/1#
title: #Plain-text with inline HTML using escaping#
link: #http://www.example.com/a/1#
description: #This is supposed to be a paragraph. With a <a href="http://www.example.com/">link inside</a>. Another paragraph, with <a href="http://www.example.com/">yet another link</a>.#
datePublished: #Sun Mar 14 19:59:12 2004#
dateUpdated: #Sun Mar 14 19:59:12 2004#
# Item end ########################
# Feed end ########################

